cvat/apps/iam/views.py

```diff
@@ -79,25 +79,19 @@ def __call__(self, request):
         return self.get_response(request)
 
 
-@method_decorator(name='post', decorator=swagger_auto_schema(
-    request_body=openapi.Schema(
-        type=openapi.TYPE_OBJECT,
-        required=[
-            'url'
-        ],
-        properties={
-            'url': openapi.Schema(type=openapi.TYPE_STRING)
+@extend_schema_view(post=extend_schema(
+    summary='This method signs URL for access to the server',
+    description='Signed URL contains a token which authenticates a user on the server.'
```

Comment:
fyi: spectacular does support docstring extraction just the same as yasg. 

this would only be useful if you wanted to split summary and description while keeping them close together.